### PR TITLE
BRO 10 more

### PR DESCRIPTION
**Pull request by Simisays**
Is Deckhints for lore reasons allowed ?

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Portal to Phyrexia
+ManaCost:9
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$ When CARDNAME enters the battlefield, each opponent sacrifices three creatures.
+SVar:TrigSac:DB$ Sacrifice | Defined$ Player.Opponent | SacValid$ Creature | Amount$ 3
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChange | TriggerDescription$ At the beginning of your upkeep, put target creature card from a graveyard onto the battlefield under your control. It's a Phyrexian in addition to its other types.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Creature | GainControl$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt:7]: AnimateSubAbility$ missing

File: forge-gui/res/tokenscripts/c_3_3_a_zombie.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Name:Golem Token
```
Comment (tool4ever) [inline forge-gui/res/tokenscripts/c_3_3_a_zombie.txt:1]: Golem?

File: forge-gui/res/cardsfolder/upcoming/ashnod_flesh_mechanist.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ManaCost:B
+Types:Legendary Creature Human Artificer
+PT:1/1
+K:Deathtouch
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigToken | OptionalDecider$ You | TriggerDescription$ Whenever CARDNAME attacks, you may sacrifice another creature. If you do, create a tapped Powerstone token.
+SVar:TrigToken:AB$ Token | Cost$ Sac<1/Creature.Other/another creature> | TokenTapped$ True | TokenScript$ c_a_powerstone
+A:AB$ Token | Cost$ 5 ExileFromGrave<1/Creature/creature card> | TokenTapped$ True | TokenScript$ c_3_3_a_zombie | SpellDescription$ Create a tapped 3/3 colorless Zombie artifact creature token.
+DeckHas:Ability$Sacrifice|Token|Graveyard & Type$Zombie|Artifact
+SVar:HasAttackEffect:TRUE
+Oracle:Deathtouch\nWhenever Ashnod, Flesh Mechanist attacks, you may sacrifice another creature. If you do, create a tapped Powerstone token.\n{5}, Exile a creature card from your graveyard. Create a tapped 3/3 colorless Zombie artifact creature token.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ashnod_flesh_mechanist.txt:11]: `graveyard.` > `graveyard:` ?

File: forge-gui/res/cardsfolder/upcoming/harbin_vanguard_animator.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Harbin, Vanguard Aviator
+ManaCost:W U
+Types:Lengendary Creature Human Soldier
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/harbin_vanguard_animator.txt:3]: Lengendary

File: forge-gui/res/cardsfolder/upcoming/harbin_vanguard_animator.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Harbin, Vanguard Aviator
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/harbin_vanguard_animator.txt:1]: Name: does not match filename

File: forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Name:Portal to Phyrexia
+ManaCost:9
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$ When CARDNAME enters the battlefield, each opponent sacrifices three creatures.
+SVar:TrigSac:DB$ Sacrifice | Defined$ Player.Opponent | SacValid$ Creature | Amount$ 3
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt:5]: `Defined$ Player.Opponent` > `Defined$ Opponent`

File: forge-gui/res/cardsfolder/upcoming/yotian_medic.txt [hunk 7]
```diff
@@ -0,0 +1,6 @@
+Name:Yotian Medic
+ManaCost:2 W
+Types:Creature Human Soldier Creature
+PT:1/4
+K:Lifelink
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/yotian_medic.txt:5]: DeckHas:Ability$LifeGain

File: forge-gui/res/cardsfolder/upcoming/yotian_dissident.txt [hunk 8]
```diff
@@ -0,0 +1,9 @@
+Name:Yotian Dissident
+ManaCost:G W
+Types:Creature Human Artificer
+PT:1/1
+T:Mode$ ChangesZone | ValidCard$ Artifact.YouCtrl  | Origin$ Any | Destination$ Battlefield | Execute$ TrigCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever an artifact enters the battlefield under your control, put +1/+1 counter on target creature you control.
+SVar:TrigCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | CounterType$ P1P1
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/yotian_dissident.txt:6]: add TgtPrompt

**Comment by Northmoc** [comment]
> Is Deckhints for lore reasons allowed ?

If there's any synergy at all it's probably fine, sure

File: forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt]: You didn't use AnimateSubAbility$

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt]: Hmm what's the difference between normal subability ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt]: It needs to ETB as phyrexian:

> 611.2e If a resolving spell or ability both puts a nontoken permanent onto the battlefield and creates
a continuous effect stating that the permanent “is [characteristic],” that continuous effect applies
simultaneously with the permanent entering the battlefield. This characteristic is usually a color
or a creature type. If the continuous effect says the permanent “becomes [characteristic]” or
“gains [an ability],” that effect applies after the permanent is on the battlefield.

You need to use `Defined$ Remembered`

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt]: Don't use RememberChanged & Cleanup, that's handled by the engine in this special case
Always check how other cards use it

File: forge-gui/res/cardsfolder/upcoming/yotian_dissident.txt [hunk 9]
```diff
@@ -2,8 +2,8 @@ Name:Yotian Dissident
 ManaCost:G W
 Types:Creature Human Artificer
 PT:1/1
-T:Mode$ ChangesZone | ValidCard$ Artifact.YouCtrl  | Origin$ Any | Destination$ Battlefield | Execute$ TrigCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever an artifact enters the battlefield under your control, put +1/+1 counter on target creature you control.
-SVar:TrigCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | CounterType$ P1P1
+T:Mode$ ChangesZone | ValidCard$ Artifact.YouCtrl | Origin$ Any | Destination$ Battlefield | Execute$ TrigCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever an artifact enters the battlefield under your control, put +1/+1 counter on target creature you control.
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/upcoming/yotian_dissident.txt:5]: put **a** +1/+1 counter....
This also goes for the Oracle text

File: forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt]: The RememberChanged$ is still there... xD

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt]: Oh I see, build in with AnimateSubAbility ...

**Review by tool4ever — APPROVED** [review]

**Pull request merged**